tests.py

```diff
@@ -831,6 +831,26 @@ def test_conan_config_install(self):
             assert remote["url"] == remote_url, "Invalid remote url"
             assert remote["verify_ssl"] == verify_ssl, "Invalid verify_ssl"
 
+    def test_conan_config_install_args(self):
+        content = textwrap.dedent("""
+            cmake_minimum_required(VERSION 3.9)
+            project(FormatOutput CXX)
+            message(STATUS "CMAKE VERSION: ${CMAKE_VERSION}")
+
+            set(CONAN_DISABLE_CHECK_COMPILER ON)
+            include(conan.cmake)
+            conan_config_install(ITEM https://github.com/conan-io/cmake-conan.git
+                                 TYPE git
+                                 ARGS -b v0.5)
+        """)
+        save("CMakeLists.txt", content)
+        os.makedirs("build")
+        os.chdir("build")
+
+        os.environ.update({"CONAN_TRACE_FILE": f"{os.getcwd()}/trace.log", "CONAN_VERBOSE_TRACEBACK": "1", "CONAN_LOGGING_LEVEL" : "40"})
+
+        run("cmake .. %s" % generator)
```

Comment:
Maybe you can check something in the output to validate that the conan config install was succesful like in other tests:

```python
        run("cmake .. {} -DCMAKE_BUILD_TYPE=Release > output.txt".format(generator))
        with open('output.txt', 'r') as file:
            data = file.read()
            assert "Repo cloned!" in data
            assert "Copying file" in data
```

Reply:
Perfect, I'll take that over :+1: